src/main/java/jalasoft/com/servlets/ListServlet.java

```diff
@@ -0,0 +1,53 @@
+package jalasoft.com.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import jalasoft.com.models.CustomDate;
+import jalasoft.com.models.Status;
+import jalasoft.com.models.Task;
+import jalasoft.com.models.ToDoListPersister;
+
+public class ListServlet extends HttpServlet {

```

Comment:
I think that you can create an interface with common methods

Reply:
If you are refering to buildTask() and buildDate() methods, I moved them to a util class TaskBuilder. Since these methods were used in other classes too.